collection-input/cesm1-le-collection.yml

```diff
@@ -40,517 +30,538 @@ data_sources:
           grid: 0.9x1.25
       case_members:
         - case: f.e11.F1850C5CN.f09_f09.001
-          sequence_order: 0
-          member_id: 0
-          has_ocean_bgc: False
+          member_id: 1
+          ctrl_branch_year: 1
 
   CTRL-SLAB-OCN:
     locations:
-      - name: GLADE
+      - name: glade
         loc_type: posix
         direct_access: True
         urlpath: /glade/collections/cdg/data/cesmLE/CESM-CAM5-BGC-LE
-      - name: HPSS
-        loc_type: hsi
-        direct_access: False
-        urlpath: /CCSM/csm/CESM-CAM5-BGC-LE
     extra_attributes:
       component_attrs:
         ocn:
           grid: POP_gx1v6
       case_members:
         - case: e.e11.E1850C5CN.f09_g16.001
-          sequence_order: 0
-          member_id: 0
-          has_ocean_bgc: True
+          member_id: 1
+          ctrl_branch_year: 1
 
   20C:
     locations:
-      - name: GLADE
+      - name: glade
         loc_type: posix
         direct_access: True
         urlpath: /glade/collections/cdg/data/cesmLE/CESM-CAM5-BGC-LE
-        exclude_dirs: ['*/b.e11.B20TRC5CNBDRD.f09_g16.001.cam.h0.TS.192001-200512.nc']
-      - name: HPSS
-        loc_type: hsi
-        direct_access: False
-        urlpath: /CCSM/csm/CESM-CAM5-BGC-LE
-        exclude_dirs: ['*/b.e11.B20TRC5CNBDRD.f09_g16.001.cam.h0.TS.192001-200512.nc']
     extra_attributes:
       component_attrs:
         ocn:
           grid: POP_gx1v6
       case_members:
         - case: b.e11.B20TRC5CNBDRD.f09_g16.001
-          sequence_order: 0
           member_id: 1
-          has_ocean_bgc: True
+          ctrl_member_id: 1
+          ctrl_experiment: CTRL
           ctrl_branch_year: 402
         - case: b.e11.B20TRC5CNBDRD.f09_g16.002
-          sequence_order: 0
           member_id: 2
-          has_ocean_bgc: True
-          ctrl_branch_year: 402
+          ctrl_member_id: 1
+          ctrl_experiment: 20C
```

Comment:
I think it is misleading to call this `ctrl_member_id` since the run it refers is not a control run. 

CMIP has conventional meta data like this
```
:parent_activity_id = "CMIP" ;
:parent_experiment_id = "esm-piControl" ;
:parent_mip_era = "CMIP6" ;
:parent_source_id = "CESM2" ;
:parent_time_units = "days since 0001-01-01 00:00:00" ;
:parent_variant_label = "r1i1p1f1" ;
:branch_method = "standard" ;
:branch_time_in_child = 674885. ;
:branch_time_in_parent = 18250. ;
```
One way to think about this is to ensure that a search comprised of the `parent_*` attributes returns the right thing. So we could have
```
parent_experiment_id
parent_member_id
```
and
```
branch_time_in_child
branch_time_in_parent
```
We could consider 
```
branch_method
```
too and would want to describe a controlled vocabulary:
"standard"
"pertlim"
....
?

Reply:
I definitely like `parent_*` and the `branch_time_in_*` variables, in addition to adding them here I'll try to find time to go back and update the CMIP catalogs used in [cesm2-marbl](https://github.com/marbl-ecosys/cesm2-marbl) as well.

Instead of `branch_method`, maybe it makes sense to also include a `pertlim` field that is 0 by default? So the `20C` LENS experiments will have `pertlim` defined in the yaml file while the the `RCP` experiments won't? Then the data table will show `pertlim=0` for the `RCP` experiments, which is an accurate statement and seems as good a way as any to indicate the members differ in where they branched from rather than by perturbations in the initial conditions. I'm open to other suggestions, but don't see any major flaws with that plan.